semeio/jobs/rft/utility.py

```diff
@@ -0,0 +1,265 @@
+    if not os.path.exists(path):
+        raise argparse.ArgumentTypeError("The path {} does not exist".format(path))
+    return path
+
+
+def load_zonemap(path):
+    if not os.path.isfile(path):
+        raise argparse.ArgumentTypeError(
+            "ZoneMap file {filename} not found!".format(filename=path)
+        )
+
+    zones_at_k_value = load_and_parse_zonemap_file(path)
+
+    return ZoneMap(zones_at_k_value)
+
+
+def parse_trajectory_line(line):
+    point = line.split()
+    if len(point) < 4 or len(point) > 5:
+        raise argparse.ArgumentTypeError(
+            "Trajectory data file not on correct format: 'utm_x utm_y md tvd <zone>' - zone is optional"
+        )
+
+    try:
+        floats = [float(v) for v in point[:4]]
+    except ValueError:
+        raise argparse.ArgumentTypeError(
+            "Error: Failed to extract data from line {line}".format(line=line)
+        )
+
+    zone = point[4].strip() if len(point) == 5 else None
+    return floats + [zone]
+
+
+def load_trajectory_points(filepath):
+    """
+    The file must be on the format of <FLOAT FLOAT FLOAT FLOAT STRING> which
+    represents <well_name day month year report_step>
+
+    Parameters
+    ----------
+    filename : string
+        Filename to open
+
+    Returns
+    -------
+    [Tuple]
+        Returns a list of tuples, where each tuple consists of 4 floats and a string/None
+    """
+
+    trajectory_points = []
+
+    filename = os.path.join(filepath)
+    if not os.path.isfile(filename):
+        raise argparse.ArgumentTypeError(
+            "Warning: Trajectory file {filename} not found!".format(filename=filename)
+        )
+
+    with open(filename, "r") as f:
+        trajectory_lines = f.readlines()
+
+    trajectory_lines = [strip_comments(l) for l in trajectory_lines]
+
+    for line in trajectory_lines:
+        if not line:
+            continue
+        point = parse_trajectory_line(line)
+        trajectory_points.append(point)
+
+    return trajectory_points
+
+
+def load_and_parse_well_time_file(filename):
+    """
+    The file must be on the format of <STRING INT INT INT INT> which
+    represents <well_name day month year report_step>
+
+    Parameters
+    ----------
+    filename : string
+        Filename to open
+
+    Returns
+    -------
+    [Tuple]
+        Returns a list of tuples with (well, datetime, report_step)
+    """
+
+    if not os.path.isfile(filename):
+        raise argparse.ArgumentTypeError("The path {} does not exist".format(filename))
+
+    well_times = []
+    base_error_msg = (
+        "Line {line_number} in well_and_time_file {filename} not on proper format: "
+    )
+
+    with open(filename) as well_time_file:
+        lines = well_time_file.readlines()
+
+    well_time_lines = [(strip_comments(l), i + 1) for i, l in enumerate(lines)]
+
+    for line, line_number in well_time_lines:
+        tokens = line.split()
+
+        if not tokens:
+            continue
+
+        elif len(tokens) != 5:
```

Comment:
This `elif` could just be an `if`i guess?

Reply:
fixed